### ISSUE TITLE
Checkbox trees can't be operated from the keyboard

### ISSUE THREAD

**Issue by eatyourgreens** [nismod/irv-frontend#67]
https://github.com/user-attachments/assets/fedf46bd-dd57-4b4e-b299-20caa8dbc777

Here, I'm tabbing to the checkbox tree, then navigating it with the cursor keys. I should be able to toggle each checkbox on/off with space (or enter) but that doesn't work. I think the keyboard handler for expanding/contracting a list is overriding the handler for the checkbox control.

I fixed this in https://github.com/nismod/irv-jamaica. I'll go back and dig out the changes that I made.

**Comment by eatyourgreens** [comment]
Here's a checkbox tree in https://github.com/nismod/irv-jamaica. I'm using the cursor keys to navigate the tree (left/right to open and close tree nodes, up/down to move through the tree) and space/enter to select/deselect the focused node. I think this is how a checkbox tree is expected to work from the keyboard.


https://github.com/user-attachments/assets/ee012b7e-3044-4fa4-9084-2fb608d9268a

**Comment by eatyourgreens** [comment]
I think this PR fixed the keyboard handlers for the tree view, as part of an upgrade to Material UI.
- https://github.com/nismod/irv-jamaica/pull/2

This PR might also be relevant: fixing missing labels for input controls. I'm not sure that unlabelled checkboxes are a problem for the checkbox trees in the global tool.
- https://github.com/nismod/irv-jamaica/pull/134
